dandischema/models.py

```diff
@@ -190,6 +207,9 @@ class Config:
         @staticmethod
         def schema_extra(schema: Dict[str, Any], model) -> None:
             schema["title"] = name2title(schema["title"])
+            if schema["type"] == "object":
+                schema["required"] = schema.get("required", [])
+                schema["required"].append("schemaKey")
```

Comment:
a matter of taste but could be 
```suggestion
                schema["required"] = schema.get("required", []) + ["schemaKey"]
```
BTW -- are we certain it will not be in "required" already somehow and we would end up with two? may be should be conditioned?